### PR TITLE
add foreman_m2 package

### PR DESCRIPTION
**Pull request by ianballou**
For plugin updates, please indicate which repos this should be built into:

* [x] Nightly
* [ ] 1.19
* [ ] 1.18
* [ ] 1.17

See Foreman's [plugin maintainer documentation](https://projects.theforeman.org/projects/foreman/wiki/How_to_Create_a_Plugin#Release-strategies) for more information.

---

### CODE REVIEW

File: packages/plugins/rubygem-foreman_m2/rubygem-foreman_m2.spec [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+%global gem_name foreman_m2
+%global plugin_name m2
+%global foreman_min_version 1.20
+
+Name: %{?scl_prefix}rubygem-%{gem_name}
+Version: 0.0.1
+Release: 1%{?foremandist}%{?dist}
+Summary: M2 bare metal provisioning plugin for Foreman
+Group: Applications/Systems
+License: GPL-3.0
```
Comment (ekohl) [inline packages/plugins/rubygem-foreman_m2/rubygem-foreman_m2.spec]: The correct Fedora license code is `GPLv3`

**Comment by stbenjam** [comment]
[test]

**Comment by ekohl** [comment]
ok to test

**Comment by stbenjam** [comment]
ok to test

**Comment by stbenjam** [comment]
ok to test

**Comment by ekohl** [comment]
[test] it failed because the tests started and then you force pushed so rekicking it.

**Pull request merged**